src/main/java/games/strategy/engine/lobby/client/ui/LobbyGamePanel.java

```diff
@@ -158,7 +160,7 @@ private void mouseOnGamesList(final MouseEvent e) {
       return;
     }
     // we sort the table, so get the correct index
-    final GameDescription description = gameTableModel.get(selectedIndex);
+    final GameDescription description = gameTableModel.get(gameTable.convertRowIndexToModel(selectedIndex));
```

Comment:
This seems like a good candidate for an Extract Method refactoring to avoid accidentally re-introducing this problem in the future.